cluster/ci/config/cluster-autoscaler.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: cluster-autoscaler
+- apiVersion: apps/v1beta1
+  kind: Deployment
+  metadata:
+    name: cluster-autoscaler
+  spec:
+    template:
+      metadata:
+        labels:
+          app: cluster-autoscaler
+          role: infra
+      spec:
+        serviceAccountName: cluster-autoscaler
+        nodeSelector:
+          role: infra
+          subrole: master
+        containers:
+        - name: autoscaler
+          image: docker.io/openshift/kubernetes-autoscaler:v0.6.1
+          args:
+          - --alsologtostderr
+          - --v=3
+          - --leader-elect-resource-lock=configmaps
+          #- --cloud-config=~/.config/gcloud/gce.ini
+          - --nodes=2:9:https://content.googleapis.com/compute/v1/projects/openshift-gce-devel/zones/us-central1-a/instanceGroups/origin-ci-ig-n-scale
```

Comment:
template this?

Reply:
Not yet, still concrete to the cluster.  In future maybe.